guideline/5.4.1.RELEASE/en/Appendix/Lombok.html

```diff
@@ -45,7 +45,7 @@ <h3>Navigation</h3>
       </ul>
     </div>
       <div class="sphinxsidebar" role="navigation" aria-label="main navigation">
-        <div class="sphinxsidebarwrapper"><label><input id="scroll" type="checkbox" checked> scroll sidebar</label>
+        <div class="sphinxsidebarwrapper"><br/><img src="../_static/logo_color.png" alt="TERASOLUNA" title="TERASOLUNA" ><br/><label><input id="scroll" type="checkbox" checked> scroll sidebar</label>
```

Comment:
- [ ] It seems `title="TERASOLUNA"` is not required.

Reply:
2631514b07201e339815dd90973e9eaa7fdd3ef8

Fixed .